tests/integ/test_stored_procedure.py

```diff
@@ -2120,3 +2120,28 @@ def upload_and_copy_into(session_):
     # sproc execution
     ingestion = sproc(upload_and_copy_into, return_type=StringType())
     assert ingestion() == "success"
+
+
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="data source is not supported in local testing",
+    run=False,
+)
```

Comment:
can you add the support to local testing?
I see is_return_table is already available in the mocking module